src/ray/common/scheduling/scheduling_ids.cc

```diff
@@ -1,7 +1,7 @@
 #include "scheduling_ids.h"
 using namespace std;
 
-int64_t StringIdMap::get(const string sid) {
+int64_t StringIdMap::get(const string& sid) {
```

Comment:
What is `sid`?

Reply:
stands for "id in string format".

Reply:
no need to be so terse here, would be more clear to call it `string_id` or `id_string`